api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -59,35 +60,38 @@ public void bufferBillingProject() {
   }
 
   public void syncBillingProjectStatus() {
-    BillingProjectBufferEntry entry = billingProjectBufferEntryDao
-        .findFirstByStatusOrderByLastSyncRequestTimeAsc(StorageEnums.billingProjectBufferStatusToStorage(CREATING));
+    for (int i = 0; i < SYNCS_PER_INVOCATION; i++) {
```

Comment:
opt: Probably a bit more efficient to just do a limit 5 on the dao, then iterate on the list in memory.

Reply:
Yeah... I did look into it but it requires adding pagination code to the repository query since Hibernate/JPA does not support LIMIT queries. Seemed like enough additional effort that wasn't worth the return.  I am also hoping to switch back to an n=1 function if the Google Tasks thing works out.